tools/lsp/ui/views/runtime-property-view.slint

```diff
@@ -0,0 +1,74 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+import { Palette, VerticalBox, ScrollView, Button }  from "std-widgets.slint";
+
+import { ExpandableGroup } from "../components/property-widgets.slint";
+
+import { Api, ColorData, ElementInformation, RuntimeComponent, RuntimeProperty, PropertyDeclaration, PropertyGroup, PropertyInformation, PropertyValue, PropertyValueKind } from "../api.slint";
+import { GroupHeader } from "../components/group.slint";
+import { EditorSizeSettings, EditorSpaceSettings } from "../components/styling.slint";
+
+import { ExpandableGroup, RuntimePropertyValueWidget } from "../components/property-widgets.slint";
+
+export component RuntimePropertyView inherits ScrollView{
+
+    property <[RuntimeComponent]> runtime-components <=> Api.runtime-components;
+
+    property <length> key-width: self.width / 2.5;
+    property <bool> element-loaded: root.runtime-components.length > 0;
+
+    content-layer := VerticalLayout {
+        if !root.element-loaded: Text {
+            text: @tr("Waiting for preview to load");
+            horizontal-alignment: center;
+            vertical-alignment: center;
+            vertical-stretch: 1;
+        }
+
+        if root.element-loaded: component := VerticalBox {
+            alignment: start;
+
+            for ep in root.runtime-components: ExpandableGroup {
+                enabled: root.enabled;
+
+                text: ep.component-name;
+                panel-width: content-layer.width;
+
+                init => {
+                    debug("INIT Expandable Group:", ep);
+                }
+
+                VerticalLayout {
+                    spacing: EditorSpaceSettings.property-spacing;
+                    padding-left: EditorSpaceSettings.group-indent;
+                    padding-right: EditorSpaceSettings.group-indent;
+                    padding-top: EditorSpaceSettings.default-padding;
+                    padding-bottom: EditorSpaceSettings.default-padding;
+
+                    for p in ep.properties: RuntimePropertyValueWidget {
+                        runtime-property: p;
+
+                        init => {
+                            debug("INIT RuntimePropertyWidget:", p);
+                        }
+                    }
+
+                    Button {
```

Comment:
i guess this is also debug that shouldn't be merged.

Reply:
I think I caught all the println's this time at least:-) I use `debug()` too rarely to remember to remove those as well.

Reply:
I think I got them all this time round.